internal/capability/report.go

```diff
@@ -67,12 +67,12 @@ func (opt OperatorInstallRptOptionFile) report(ca CapAudit) error {
 	}
 	defer file.Close()
 
-	if !ca.CsvTimeout {
+	if !ca.csvTimeout {
 
-		file.WriteString("{\"level\":\"info\",\"message\":\"" + string(ca.Csv.Status.Phase) + "\",\"package\":\"" + ca.Subscription.Package + "\",\"channel\":\"" + ca.Subscription.Channel + "\",\"installmode\":\"" + string(ca.Subscription.InstallModeType) + "\"}\n")
+		file.WriteString("{\"level\":\"info\",\"message\":\"" + string(ca.csv.Status.Phase) + "\",\"package\":\"" + ca.subscription.Package + "\",\"channel\":\"" + ca.subscription.Channel + "\",\"installmode\":\"" + string(ca.subscription.InstallModeType) + "\"}\n")
```

Comment:
Rather than open a file here, this method should probably accept an io.Writer and just write to that.

Reply:
f1a3a82